README.md

```diff
@@ -13,7 +13,7 @@ To get up and running within the WooCommerce Monorepo, you will need to make sur
 ### Prerequisites
 
 -   [NVM](https://github.com/nvm-sh/nvm#installing-and-updating): While you can always install Node through other means, we recommend using NVM to ensure you're aligned with the version used by our development teams. Our repository contains [an `.nvmrc` file](.nvmrc) which helps ensure you are using the correct version of Node.
--   [PNPM](https://pnpm.io/installation): Our repository utilizes PNPM version 9.1.3 to manage project dependencies and run various scripts involved in building and testing projects.
+-   [PNPM](https://pnpm.io/installation): Our repository utilizes PNPM to manage project dependencies and run various scripts involved in building and testing projects.
```

Comment:
haha kinda funny we specified the version here XD

Reply:
Yeah, that iteration of fixing the  pnpm version was not great. This PR makes fair step forward.